packages/coil-extension/src/background/services/BackgroundScript.ts

```diff
@@ -712,7 +716,10 @@ export class BackgroundScript {
     const id = this.assoc.getStreamId(frame)
     if (id) {
       this.log('resuming stream', id)
-      this.sendSetMonetizationStateMessage(frame, 'pending')
+      this.sendSetMonetizationStateMessage(
+        frame,
+        this.hasStreamPaid(frame) ? 'started' : 'pending'
```

Comment:
So we decided with sending started immediately in this case

Reply:
Yes, I think that's best. It is resuming payment, and has successfully paid already.

Reply:
I think we should send both to be consistent